src/generateOutsideCheck.js

```diff
@@ -1,4 +1,21 @@
 /**
+ * Check if classList contains any of the given classes,
+ * transparently handling singular and array cases.
+ */
+function containsAny(classList, classes) {
+  if (Array.isArray(classes)) {
+    for (const className of classes) {
```

Comment:
`for-of` loop could be avoided here (its creating quite [weirdo code after transpilation](https://babeljs.io/repl/#?babili=false&browsers=ie%2011&build=&builtIns=false&circleciRepo=&code_lz=GYVwdgxgLglg9mABBBUCGMwGcCCYCeAFBADZpZYAyMWUANMmRQKZYCUiA3gFCKIzBEhHACcRafADoao8UVLksrNhx58-wOCKEpsURooByaALbNEcQQpbsuvdXwE6mVGlEm70mLMRfGzKnYODiLMUCAiSFAiIMwA3PYOAL6JiCkhYRFIwGgkSgl8SYjMeeZq6qHhkQYU1LQeqBjYvorKBWncSUA&debug=false&evaluate=false&lineWrap=false&presets=es2015&prettier=false&showSidebar=true&targets=&version=6.26.0), although we use `loose` mode and its slightly better than what I have presented) and it could be handled by easy `Array.prototype.some` call